atom/browser/net/url_request_stream_job.cc

```diff
@@ -0,0 +1,206 @@
+// Copyright (c) 2017 GitHub, Inc.
```

Comment:
can we make use of `content::StreamURLRequestJob` here?

Reply:
I don't know enough about the codebase to answer.